modules/tensor_mechanics/test/tests/czm/tests

```diff
@@ -0,0 +1,144 @@
+[Tests]
+  [./czm_3DC_load_Normal]
+    type = Exodiff
+    input = czm_3DC_load_only_normal.i
+    exodiff = czm_3DC_load_only_normal_out.e
+    issues = '#11546'
+    requirement = 'The system shall allow for cohesive zone laws to'
+    'represent the traction-separation behavior at an interface between'
+    'two bodies represented by continuum elements in 3D. By construction of the test,'
+    'only a normal gap should appear. The maximum theoretical '
+    'normal traction of 100 MPa shall be seen when the normal gap reaches a value of 1'
+    mesh_mode = 'REPLICATED'
+    recover = false
+    method = '!dbg' # issue 12033
+    design = 'source/materials/czm/SalehaniIrani3DCTraction.md source/interfacekernels/CZMInterfaceKernel.md'
+  [../]
+  [./czm_3DC_load_shear_x]
+    type = Exodiff
+    input = czm_3DC_load_shear_x.i
+    exodiff = czm_3DC_load_shear_x_out.e
+    issues = '#11546'
+    requirement = 'The system shall allow for cohesive zone laws to'
+    'represent the traction-separation behavior at an interface between'
+    'two bodies represented by continuum elements in 3D. By construction of the test,'
+    'only a shear gap on the x direction should appear. The maximum theoretical '
+    'shear traction of 70 MPa shall be seen when the shear gap reaches a value of 0.5'
+    mesh_mode = 'REPLICATED'
+    recover = false
+    method = '!dbg' # issue 12033
+    design = 'source/materials/czm/SalehaniIrani3DCTraction.md source/interfacekernels/CZMInterfaceKernel.md'
+  [../]
+  [./czm_3DC_load_shear_y]
+    type = Exodiff
+    input = czm_3DC_load_shear_y.i
+    exodiff = czm_3DC_load_shear_y_out.e
+    issues = '#11546'
+    requirement = 'The system shall allow for cohesive zone laws to'
```

Comment:
The system shall allow for cohesive zone laws to represent the traction-separation behavior at an interface between two bodies represented by continuum elements in 3D using the Salehani Irani 3DC model, and only compute a nonzero y-component of the tangential gap under purely shear loading in the y-direction.